Managers/SettingsMan.h

```diff
@@ -1,679 +1,558 @@
+		bool GetServerTransmitAsBoxes() const { return m_ServerTransmitAsBoxes; }
+
+		/// <summary>
+		/// Gets the width of the transmitted block when transmitting frames as blocks.
+		/// </summary>
+		/// <returns>The width of the transmitted block.</returns>
+		unsigned short GetServerBoxWidth() const { return m_ServerBoxWidth; }
+
+		/// <summary>
+		/// Gets the height of the transmitted block when transmitting frames as blocks.
+		/// </summary>
+		/// <returns>The height of the transmitted block.</returns>
+		unsigned short GetServerBoxHeight() const { return m_ServerBoxHeight; }
+
+		/// <summary>
+		/// Gets whether a NAT service is used for punch-through.
+		/// </summary>
+		/// <returns>Whether a NAT service is used for punch-through or not.</returns>
+		bool GetUseNATService() { return m_UseNATService; }
+
+		/// <summary>
+		/// Gets whether server puts threads to sleep if it didn't receive anything for 10 seconds to reduce CPU load.
+		/// </summary>
+		/// <returns>Whether threads will be put to sleep when server isn't receiving any data or not.</returns>
+		bool GetServerSleepWhenIdle() { return m_ServerSleepWhenIdle; }
+
+		/// <summary>
+		/// Gets whether the server will try to put the thread to sleep to reduce CPU load if the sim frame took less time to complete than it should at 30 fps.
+		/// </summary>
+		/// <returns>Whether threads will be put to sleep if server completed frame faster than it normally should or not.</returns>
+		bool GetServerSimSleepWhenIdle() { return m_ServerSimSleepWhenIdle; }
+#pragma endregion
+
+#pragma region Editor Settings
+		/// <summary>
+		/// Returns the list of visible assembly groups.
+		/// </summary>
+		/// <returns>List of visible assembly groups.</returns>
+		std::list<std::string> GetVisibleAssemblyGroupsList() const { return m_VisibleAssemblyGroupsList; }
+
+		/// <summary>
+		/// Whether editors will allow to select Base.rte as a module to save in
+		/// </summary>
+		/// <returns>True of editors are allowed to select Base.rte as a module to save in.</returns>
+		bool AllowSavingToBase() const { return m_AllowSavingToBase; }
+
+		/// <summary>
+		/// Whether we need to show MetaScenes in editors and scenario UI.
+		/// </summary>
+		/// <returns>True if we need to show MetaScenes.</returns>
+		bool ShowMetascenes() const { return m_ShowMetaScenes; }
+#pragma endregion
+
+#pragma region Mod Manager
+		/// <summary>
+		/// Returns map of mods which were disabled.
+		/// </summary>
+		/// <returns>Map of mods which were disabled.</returns>
+		std::map<std::string, bool> GetDisabledModsList() const { return m_DisabledMods; }
+
+		/// <summary>
+		/// Adds specified mod to internal list of disabled mods.
+		/// </summary>
+		/// <param name="modModule">Mod to enable.</param>
+		void EnableMod(std::string modModule) { std::transform(modModule.begin(), modModule.end(), modModule.begin(), ::tolower); m_DisabledMods[modModule] = false; }
```

Comment:
No you need these transforms otherwise it screws up and shows the line as an unknown mod or whatever it was, even with the lines being all lowercase already.
I removed the transforms in ReadProperty because it was pointless to do them twice.

Reply:
Okay gotcha. Cool.